odl/tomo/backends/astra_cpu.py

```diff
@@ -36,12 +36,14 @@
                                            astra_volume_geometry, astra_data,
                                            astra_projector, astra_algorithm)
 from odl.tomo.geometry import Geometry
+from odl.tomo.geometry.fanbeam import FanFlatGeometry
 
 __all__ = ('astra_cpu_forward_projector_call',
            'astra_cpu_backward_projector_call')
 
 
 # TODO: Fix inconsistent scaling of ASTRA projector with pixel size
```

Comment:
Hmm. I hoped so and thus it should have been removed, but as Jonas realized I forgot to check non-isotropic pixels. So it's maybe still an issue.